doc/nrf/protocols/matter/getting_started/custom_clusters.rst

```diff
@@ -0,0 +1,408 @@
+           <attribute side="server" code="0x00" define="FLIP_FLOP" type="boolean" writable="true" default="false" optional="false">MyAttribute</attribute>
+           <command source="client" code="0x02" name="MyCommand" response="MyCommandResponse" optional="false">
+             <description>Command that takes two uint8 arguments and returns their sum.</description>
+             <arg name="arg1" type="int8u"/>
+             <arg name="arg2" type="int8u"/>
+           </command>
+           <event source="server" code="0x01" name="MyEvent" optional="false">
+             <description>Event that is generated by the server.</description>
+             <arg name="arg1" type="int8u"/>
+           </event>
+         </cluster>
+
+   .. tab:: ``<deviceType>``
+
+      ``<deviceType>`` elements define the device types that can be used in the cluster and consist of the following:
+
+      * ``<name>`` - The unique name of the device.
+      * ``<domain>`` - The domain to which the device belongs.
+      * ``<typeName>`` - The name of the device displayed in the zap tool.
+      * ``<profileId>`` - The profile ID of the device.
+
+         * ``editable`` - Indicates if the field can be modified.
+
+      * ``<deviceId>`` - The device ID.
+
+         * ``editable`` - Indicates if the field can be modified.
+
+      * ``<class>`` - The class of the device.
+         Can be ``Utility``, ``Simple``, or ``Node``.
+      * ``<scope>`` - The scope of the device.
+         Can be ``Node``, or ``Endpoint``.
+      * ``<clusters>`` - The definition of an individual item within the structure.
+
+         * ``lockOthers`` - Indicates if other clusters are locked.
+         * ``<include>`` - Defines a cluster that should be included in the device.
+
+            * ``cluster`` - The name of the cluster.
+            * ``client`` - Indicates if the client role should be enabled.
+            * ``server`` - Indicates if the server role should be enabled.
+            * ``clientLocked`` - Indicates if the client role modification should be locked.
+            * ``serverLocked`` - Indicates if the server role modification should be locked.
+            * ``<requireAttribute>`` - Defines a required attribute name.
+            * ``<requireCommand>`` - Defines a required command name.
+
+      For example, the following XML code defines a structure with one item of type octet string and length 128:
+
+      .. code-block:: xml
+
+         <deviceType>
+           <name>my-new-device</name>
+           <domain>CHIP</domain>
+           <typeName>My new device</typeName>
+           <profileId editable="false">0x0FFF</profileId>
+           <deviceId editable="false">0x001</deviceId>
+           <class>Simple</class>
+           <scope>Endpoint</scope>
+           <clusters lockOthers="true">
+           <include cluster="MyNewCluster" client="true" server="true" clientLocked="false" serverLocked="false"/>
+             <requireAttribute>FLIP_FLOP</requireAttribute>
```

Comment:
```suggestion
             <requireAttribute>MyAttribute</requireAttribute>
```

Reply:
requireAttribute requires define, not name. but will change `MyAttribute`'s define from `FLIP_FLOP` to `MY_ATTRIBUTE`